spark/common/src/main/scala/org/apache/sedona/sql/UDF/PythonEvalType.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.sedona.sql.UDF
+
+object PythonEvalType {
+  val SQL_SCALAR_SEDONA_UDF = 5200
```

Comment:
Let's add comment to clarify that 5200 is `SEDONA_UDF_TYPE_CONSTANT` + `SQL_SCALAR_PANDAS_UDF`, where `SQL_SCALAR_PANDAS_UDF` is 200.